### DISCUSSION TITLE
Can we use the Microsoft Graph data source in production now?

### DISCUSSION THREAD

**Discussion by mariaarcena** [microsoft-search/pnp-modern-search#1506] [Q&A] [answered]
Hello everyone! I was wondering if we can use the Microsoft Search as a data source since I see in the recent releases that there has been made improvements. But in the official document website, it still says that it is not recommended to use Microsoft Graph because of it being in beta or using the beta version?

May someone guide me on this? :)

**Comment by wobba** [comment, marked as answer]
Sure you can and we can update the docs. If your scenario is pure SP content I personally still prefer the SP API. But it depends on the scenario.
